src/packet_parser/packet_parser.go

```diff
@@ -0,0 +1,104 @@
+package packet_parser
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+
+	excelAdapterModels "github.com/HyperloopUPV-H8/Backend-H8/excel_adapter/models"
+	"github.com/HyperloopUPV-H8/Backend-H8/packet_parser/internals"
+	"github.com/HyperloopUPV-H8/Backend-H8/packet_parser/models"
+)
+
+type PacketParser struct {
+	descriptors map[uint16]models.PacketDescriptor
+	enums       map[string]models.Enum
+}
+
+func (parser *PacketParser) AddPacket(board string, ip string, desc excelAdapterModels.Description, values []excelAdapterModels.Value) {
+	if parser.descriptors == nil {
```

Comment:
Possible error should be handled outside this function.

Reply:
and where should it be?